src/guild_configs/lbp.js

```diff
@@ -180,618 +180,665 @@ const lbp = {
+                "Lead the Weigh": null,
+                "Fight And Flight": null,
+                "Science Friction": null,
+                "Pros And Conveyors": null,
+                "One Track Mind": null,
+                "This Way Up": null,
+                "Escape Velocity": null,
+                "Swipe Right": null,
+                "Nervous System": null,
+                "Spaceport Dash": null,
+
+
+                "The Struggle Is Rail": null,
+                "Flossed In Space": null,
+
+
+                "Off The Rails": null,
+                "Keep It Tidy": null,
+                "Double Down": null,
+                "Stick or Twist": null,
+                "Flash Forward": null,
+                "Just A Phase": null,
+                "Crate Expectations": null,
+                "Multitask Force": null,
+                "Doom & Bloom": null,
+                "Until Vex Time": null,
+                "Doom & Dash": null,
+                "Jumping to Conclusions": null,
+                "Vexpiration Date": null,
+
+
+                "Keep On Leaping On": null,
+                "Best in Throw": null,
+                "High and Mighty": null,
+                "In Full Swing": null,
+                "Little Big Finale": { aliases: [ "finale" ] },
+            },
+        },
+        "DLC": {
+            aliases: [ "lbpdlc", "lbpseriesdlc", "seriesdlc" ],
+            categories: {
+                "Individual Levels": {
+                    aliases: ilAliases,
+                    default: true,
+                    il: true,
+                },
+            },
+            levels: {
+                "Metal Gear Solid Level Pack": {
+                    aliases: [ "mgs" ],
+                    default: true,
+                },
+                "Pirates of the Caribbean Level Kit": { aliases: [ "potc" ] },
+                "Marvel Level Pack": null,
+                "Toy Story Level Kit": null,
+                "Move Pack: The Rise of the Cakeling": { aliases: [ "rotc" ] },
+                "The Muppets Level Kit": null,
+                "Cross-Controller Pack": { aliases: [ "ccp", "xcp", "cc" ] },
+                "DC Comics Level Pack": { aliases: [ "dc" ] },
+                "DC Comics Level Pack (Vita)": { aliases: [ "dccomicsv", "dcvita", "dcv" ] },
+                "LittleBigPlanet 3: The Journey Home": { aliases: [ "journeyhome" ] },
+
+
+                "Monsters Kit": { aliases: [ "monster" ] },
+                "History Kit": null,
+                "The Incredibles Level Kit": null,
+                "The Nightmare Before Christmas Level Kit": { aliases: [ "nbc" ] },
+                "Tiki Paradise Level Kit": { aliases: [ "tiki" ] },
+                "Adventure Time Level Kit": null,
+                "Back to the Future Level Kit": null,
+                "Seaside Surprise Level Kit": { aliases: [ "seaside" ] },
+
+
+                "Act 1: Introduction": { aliases: [ "act1", "intro", "introduction" ] },
```

Comment:
this shouldn't be necessary because "introduction" is automatically replaced with "intro" in `lbpCleanUpLevelName`